src/components/elements/buttons/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Arrow from '../../../../static/svg/arrow'
+import { font, size, letterSpacing, colors } from '../../../utils'
+
+const StyledButton = styled.button`
+  height: 5rem;
+  ${font};
+  font-size: ${size.xxs};
+  font-weight: 500;
+  letter-spacing: ${letterSpacing.medium};
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem 2rem;
+  border-radius: 5px;
+  border: none;
+  /* props */
+  color: ${({ textColor }) => textColor};
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  text-transform: ${({ capitalize }) => (capitalize ? 'uppercase' : 'none')};
+`
+const StyledArrow = styled(Arrow)`
+  display: flex;
+  margin-left: 1rem;
+  path {
+    fill: ${({ color }) => color};
+  }
+`
+const Button = ({
+  title,
+  hasArrow,
+  textColor,
+  backgroundColor,
+  onClick,
+  capitalize,
+  primary,
+}) => {
+  const bgColor = (primary && colors.primary) || backgroundColor
+  return (
```

Comment:
the onClick event is required you can add it like so : 
 ```suggestion
const Button = ({
  title,
  hasArrow,
  textColor,
  backgroundColor,
  onClick,
  capitalize,
  primary,
}) => {
  const bgColor = (primary && colors.primary) || backgroundColor
  const onClik = onClick || () => alert('you must add onClick event')
  return (
```